src/datepicker/themes/bs/bs-datepicker-view.html

```diff
@@ -6,19 +6,32 @@
     <!--calendars-->
     <div class="bs-calendar-container" [ngSwitch]="viewMode | async" role="application">
       <!--days calendar-->
-      <div *ngSwitchCase="'day'" class="bs-media-container">
-        <bs-days-calendar-view
-          *ngFor="let calendar of daysCalendar$ | async"
-          [class.bs-datepicker-multiple]="multipleCalendars"
-          [calendar]="calendar"
-          [options]="options$ | async"
-          (onNavigate)="navigateTo($event)"
-          (onViewMode)="setViewMode($event)"
-          (onHover)="dayHoverHandler($event)"
-          (onHoverWeek)="weekHoverHandler($event)"
-          (onSelect)="daySelectHandler($event)">
-        </bs-days-calendar-view>
-      </div>
+      <ng-container *ngSwitchCase="'day'">
+        <div class="bs-media-container">
+          <bs-days-calendar-view
+            *ngFor="let calendar of daysCalendar$ | async"
+            [class.bs-datepicker-multiple]="multipleCalendars"
+            [calendar]="calendar"
+            [options]="options$ | async"
+            (onNavigate)="navigateTo($event)"
+            (onViewMode)="setViewMode($event)"
+            (onHover)="dayHoverHandler($event)"
+            (onHoverWeek)="weekHoverHandler($event)"
+            (onSelect)="daySelectHandler($event)">
+          </bs-days-calendar-view>
+        </div>
+        <div *ngIf="withTimepicker" class="bs-timepicker-in-datepicker-container">
+          <timepicker
+            [ngModel]="(selectedTime | async)![0]"
```

Comment:
this looks extremely weird, plus you will need to expose timepicker value without forms usage

Reply:
replaced `ngModel` with `writeValue` and `registerOnChange`